demo/fields.py

```diff
@@ -1,9 +1,9 @@
 import json
 
-from wagtail.fields import StreamField as WagtailStreamfield
+from wagtail.fields import StreamField as WagtailStreamField
 
 
-class StreamField(WagtailStreamfield):
+class StreamField(WagtailStreamField):
```

Comment:
Curious to know what this override really is for, I don't understand why a demo project needs something like this 🤔

Reply:
Oh, it's just to avoid migration noise from changes in the StreamField blocks definition. Originally I expected to be adding several block types in sequence to demo their features.